OOVPADatabase/D3D8LTCG.OOVPA.inl

```diff
@@ -112,30 +114,33 @@ OOVPATable D3D8LTCG_OOVPAV2[] = {
 	REGISTER_OOVPAS(D3DDevice_LightEnable, 1024, 1036),
 	REGISTER_OOVPAS(D3DDevice_LightEnable_4, 2024, 2048),
 	REGISTER_OOVPAS(D3DDevice_LoadVertexShaderProgram, 1024, 1036),
+	REGISTER_OOVPAS(D3DDevice_LoadVertexShaderProgram_4, 2048),
 	REGISTER_OOVPAS(D3DDevice_LoadVertexShader_0, 2024, 2036),
 	REGISTER_OOVPAS(D3DDevice_LoadVertexShader_4, 2024, 2048),
 	REGISTER_OOVPAS(D3DDevice_MultiplyTransform_0, 2024),
-	REGISTER_OOVPAS(D3DDevice_PersistDisplay, 1048, 1060),
+	REGISTER_OOVPAS(D3DDevice_PersistDisplay, 1024, 1048, 1060),
 	REGISTER_OOVPAS(D3DDevice_Present, 1024),
 	REGISTER_OOVPAS(D3DDevice_Release, 1024),
 	REGISTER_OOVPAS(D3DDevice_Reset, 1024, 1036),
 	REGISTER_OOVPAS(D3DDevice_Reset_0, 2024),
+	REGISTER_OOVPAS(D3DDevice_RunPushBuffer, 1024, 1036, 1048),
 	REGISTER_OOVPAS(D3DDevice_RunVertexStateShader_4, 2048),
-	REGISTER_OOVPAS(D3DDevice_SelectVertexShader, 1024),
-	REGISTER_OOVPAS(D3DDevice_SelectVertexShader_0, 2048, 2060),
+	REGISTER_OOVPAS(D3DDevice_SelectVertexShader, 1024, 1036),
+	REGISTER_OOVPAS(D3DDevice_SelectVertexShader_0, 2024, 2048, 2060),
 	REGISTER_OOVPAS(D3DDevice_SelectVertexShader_4, 2024, 2048, 2060),
+	REGISTER_OOVPAS(D3DDevice_SelectVertexShaderDirect_0, 2048),
 	REGISTER_OOVPAS(D3DDevice_SetFlickerFilter_0, 2024),
 	REGISTER_OOVPAS(D3DDevice_SetGammaRamp, 1024),
 	REGISTER_OOVPAS(D3DDevice_SetIndices, 1024, 1036),
 	REGISTER_OOVPAS(D3DDevice_SetIndices_4, 2024),
 	REGISTER_OOVPAS(D3DDevice_SetLight, 1024, 1048),
-	REGISTER_OOVPAS(D3DDevice_SetPalette, 1024, 1036),
+	REGISTER_OOVPAS(D3DDevice_SetPalette, 1024, 1036, 1048),
 	REGISTER_OOVPAS(D3DDevice_SetPalette_4, 2024),
 	REGISTER_OOVPAS(D3DDevice_SetPixelShader, 1024, 1036, 1048),
 	REGISTER_OOVPAS(D3DDevice_SetPixelShaderConstant, 1024, 1036),
 	REGISTER_OOVPAS(D3DDevice_SetPixelShaderConstant_4, 2024),
 	REGISTER_OOVPAS(D3DDevice_SetPixelShaderProgram, 1024),
-	REGISTER_OOVPAS(D3DDevice_SetPixelShader_0, 2024, 2036, 2048),
+	REGISTER_OOVPAS(D3DDevice_SetPixelShader_0, 2024, 2036, 2048, 2060, 2072),
```

Comment:
@jarupxx, I can't believe I missed this. Where is D3DDevice_SetPixelShader_0_2060 OOVPA at? I don't see it in this pull request.

Reply:
It was 3911 that leave it in this pull request. I wish I had been more careful.